.github/workflows/publish_ci.yml

```diff
@@ -0,0 +1,61 @@
+name: Publish to GitHub
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
@xperiandri even if we only want to publish for master branch, the way to filter it is to prevent the last step from happening (the nuget push) if branch is not master. This way the job is still being tested everytime, even for PRs.